docs/configuration.md

```diff
@@ -9,6 +9,11 @@
   - `downloads` - directory used to stage downloaded archives during Boards/Library Manager installations.
   - `user` - the equivalent of the Arduino IDE's ["sketchbook" directory][sketchbook directory]. Library Manager
     installations are made to the `libraries` subdirectory of the user directory.
+  - `builtin.libraries` - the libraries in this directory will be available to all platforms without the need for the
+    user to install them, but with the lowest priority over other installed libraries with the same name, it's the
+    equivalent of the Arduino IDE's bundled libraries directory.
+  - `builtin.tools` - it's a list of directories of tools that will be available to all platforms without the need for
+    the user to install them, it's the equivalent of the Arduino IDE's bundled tools directory.
```

Comment:
```suggestion
    the user to install them, it's the equivalent of the Arduino IDE 1.x bundled tools directory.
```
I think it is worth clarifying this since some readers will likely not be familiar with this "bundled tools directory". They might otherwise be confused after trying to learn about it by studying Arduino IDE 2.x, which has no such thing.